app/components/shared/dashboard_component/dashboard_component.scss

```diff
@@ -9,25 +9,19 @@
 
   @include govuk-media-query ($from: desktop) {
     .govuk-width-container {
-      position: relative;
+      display: flex;
     }
 
     &__button {
-      bottom: govuk-spacing(3);
-      margin-top: 0;
-      position: absolute;
-      right: 0;
+      flex-grow: 1;
+      text-align: right;
```

Comment:
can you add a `margin-bottom` to the button?

Reply:
i just pushed a different chnage for this

Reply:
removing the top margin below brings this into alignment

Reply:
I think on the template there are margin top/bottom 0 classes, so might need to change those

Reply:
they are needed

Reply:
the margin top one can be removed and i removed it